www/shared/test/utils/render-with-intl/render-with-intl.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
We don’t need to test the test utils.. instead I would ignore the test utils folder altogether in our jest config. There’s an option for this, see https://jestjs.io/docs/en/configuration#coveragepathignorepatterns-arraystring.

Moreover, we can then simplify the file structure and remove the folder as it will contain just the actual file.

Reply:
Could you please expand on why testing this is unnecessary? I'd agree if this were an external library, or otherwise code that is likely tested elsewhere, but this tool is of our own creation, and i think this should be tested to establish trust in this util.

Reply:
After conversation with @satazor, he agrees with keeping these tests.